client/a8c-for-agencies/components/sidebar-menu/sites.tsx

```diff
@@ -18,7 +21,42 @@ type Props = {
 export default function ( { path }: Props ) {
 	const translate = useTranslate();
 
+	const isPartnerOAuthTokenLoaded = useSelector( getIsPartnerOAuthTokenLoaded );
+
+	const { data, isLoading } = useFetchDashboardSites(
+		isPartnerOAuthTokenLoaded,
+		'',
+		1,
+		{
+			issueTypes: [],
+			showOnlyFavorites: false,
+		},
+		{
+			field: '',
+			direction: '',
+		}
+	);
```

Comment:
I'm curious what you guys think about this. I'm honestly not sure if this is the best approach here.

Reply:
I'm not too sure either at this stage, but getting that data separately does result in better data than the check made in the sites dashboard (see my other comment).